android/app/src/main/AndroidManifest.xml

```diff
@@ -25,8 +29,11 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-    </application>
-
-	<uses-permission android:name="android.permission.INTERNET" />
+        <service
+            android:name=".services.LocationService"
+            android:foregroundServiceType="location"
+            >
 
+        </service>
+    </application>
```

Comment:
I remember at one point we added perms for unsecure HTTP connections, did we get rid of that?

Reply:
I think we did that a long time ago before we figured out how to use OpenApi stuff.
I don't see anything about it though....

Reply:
The thing is we still had it with OpenAPI, because for a long time we were using HTTP on the direct server ip 207.X.Y.Z and it suddenly disappeared.

Reply:
Hmm took a look and can't find it, guessing we took that out when we switched to HTTPS and I forgot about the change. Just wondering :)

Reply:
Resolved in https://github.com/andr-dev/CS446/pull/82